### DISCUSSION TITLE
restart frequency control for HRLDAS needs improvement

### DISCUSSION THREAD

**Discussion by cenlinhe** [NCAR/hrldas#86] [General]
Currently, if users set RESTART_FREQUENCY_HOURS <=0, then HRLDAS will output restart file on the first day of each month. However, we should improve this so that if RESTART_FREQUENCY_HOURS = 0, then no restart file will be output:
https://github.com/NCAR/hrldas/blob/master/hrldas/IO_code/module_NoahMP_hrldas_driver.F#L1305

**Comment by tslin2** [comment]
I have also noticed this recently
